sormas-backend/src/main/java/de/symeda/sormas/backend/infrastructure/area/AreaFacadeEjb.java

```diff
@@ -125,50 +132,38 @@ public AreaDto save(@Valid AreaDto dto, boolean allowMerge) {
 		}
 
 		area = fromDto(dto, area, true);
-		areaService.ensurePersisted(area);
+		service.ensurePersisted(area);
 		return toDto(area);
 	}
 
 	@Override
-	public boolean isUsedInOtherInfrastructureData(Collection<String> areaUuids) {
-		return areaService.isUsedInInfrastructureData(areaUuids, Region.AREA, Region.class);
-	}
-
-	@Override
-	public void archive(String areaUuid) {
-		Area area = areaService.getByUuid(areaUuid);
-		area.setArchived(true);
-		areaService.ensurePersisted(area);
+	public void dearchive(String uuid) {
+		super.dearchiveUnchecked(uuid);
```

Comment:
I don't think you need to override this here, the missing check for `EDIT_INFRASTRUCTURE_DATA` essentially looks like a bug to me (i.e. it should be there)

Reply:
good catch